### DISCUSSION TITLE
Audio detector seems to prevent the object detector

### DISCUSSION THREAD

**Discussion by EM-Tom** [blakeblackshear/frigate#9024] [Ask A Question] [answered]
Hey all, 

QQ hopfully - 

I am running both audio and video detectors with great success (using the F+ model for video) . All awesome. 

What I have noticed though is since recently adding the audio detector, if someone walks by and is speech is detected I get an audio event as expected, by no person object detection event. If someone walks past quietly I get one every time without fail. Is this expected behavior? 

Thanks

**Comment by NickM-27** [comment, marked as answer]
The logic for these detections is entirely separate so it seems very unlikely that this is the case. The only way to know for sure would be to have the debug live view open and see what happens while this occurs.

I would suggest most likely the threshold in the config may be a bit too high and sometimes person events are not created.

**Comment by EM-Tom** [comment]
Cheers Nick, I'll do some debugging - glad to know it's not expected. 
I am guessing confirmation bias from myself.

Thanks
